app/routes/tasks.py

```diff
@@ -0,0 +1,128 @@
+import requests
+import os
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, abort, make_response, request
+
+bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+def validate_model(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except:
+        abort(make_response(
+            {"message": f"{cls.title} {model_id} invalid"}, 400))
+
+    model = cls.query.get(model_id)
+
+    if not model:
+        abort(make_response(
+            {"message": f"{cls.title} {model_id} not found"}, 404))
```

Comment:
I think you want `cls.__name__` here. This will print something unexpected like "Model.title", which I don't think is what you want.